<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogo_NavegacionRegistros</class>
 <widget class="QDialog" name="Dialogo_NavegacionRegistros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sqlite3 - Navegacion registro</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>322</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_id">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_id">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_nombre">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_nombre">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_primero">
        <property name="text">
         <string>Primero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_anterior">
        <property name="text">
         <string>Anterior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_siguiente">
        <property name="text">
         <string>Siguiente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ultimo">
        <property name="text">
         <string>Ãšltimo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>21</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
